src/style.css

```diff
@@ -1 +1,26 @@
-/* styles go here */
+.wrapper {
+  display: block;
+}
+
+.field {
+  margin-bottom: 20px;
+}
+
+.wrapper:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+.submitBtn {
+  margin-top: 8px;
+}
```

Comment:
No need to play with pixels, task requirement said about particular styles that should be set, other styles should be default
![Screenshot 2023-01-26 at 16 02 09](https://user-images.githubusercontent.com/84275496/214855163-1fbc38de-6f0f-4bb9-acdb-a0bacfbbd587.png)